docs/ServingTheServer.md

```diff
@@ -0,0 +1,138 @@
+# Linking Your Dev Server to Slack
+
+So, you want to see some progress, preferably on slack, and not just in the
+forms of unit testing? At this point, fear is actually a reasonable response.
+With this guide, you can be talking to your locally-hosted slack bot in no time!
+
+## 1: Registering Your Slack Bot
+
+First things first, you must first create a slack workspace (an existing one
+would do, as long as you can install things onto it).
+
+After that, you must create a new slack app. Follow the link
+[here][make-slack-app] to create a new slack app. Make sure to have it installed
+onto your slack workspace of choice.
+
+Now, you need to create a bot user. [Create it here][make-bot].
+
+## 2: Download and Start Up DynamoDB
+
+Since we are developing locally, we must use the local version of DynamoDB.
+Download the `jar` from [here][dynamodb-download]. Follow the instructions from
```

Comment:
`dynamodb-download` link doesn't exist (looks like you named it `dynamodb` below).